sdk/messaging/azservicebus/CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contributing
+
+## Getting Started
+
+First of all, welcome. Thank you for your interest in furthering Go support for Azure Service Bus.
+
+### Workstation Requirements
+
+The following programs should be installed on your machine before you begin developing.
+
+> Note: Adhering to the linters below is enforced in CI. It is not required to have the tools locally, but contributors 
+are expected to fix those issues found in CI.
+
+| Tool | Necessary | Description |
+| :---------------------------------------------------: | :-----: | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [go](https://godoc.org)                               | Required |This is a Go project, and as such you should have Go installed on your machine. We use modules for package management, so we recommend at least using go1.11. However, you may also use go1.9.7+ or go1.10.3+. |
+| [git](https://git-scm.com)                            | Required |azure-service-bus-go uses Git as its source control management solution.                                                                                                            |
+| [az](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest) | Optional | The Azure CLI is used for its ability to authenticate against Azure. Tests themselves only need a connection string and other metadata about the Service Bus namespace, but tooling to automatically setup the necessary infrastructure to run the tests may depend on the Azure CLI. |
+| [terraform](https://terraform.io)                     | Optional | `terraform` is used to help provision the Azure infrastructure needed to run our tests both in CI and on your local machine. If you have already provisioned a Service Bus Namespace, and created the necessary Service Bus Queues, you do not need terraform. |
+| [golint](https://godoc.org/golang.org/x/lint/golint)  | Optional |`golint` is a linter that finds basic stylistic mistakes in Go programs.                                                                                                            |
+| [gocyclo](https://github.com/fzipp/gocyclo)           | Optional |`gocyclo`  checks for programmatic complexity, to ensure code readability.                                                                                                          |
+| [megacheck](https://honnef.co/go/tools/cmd/megacheck) | Optional | `megacheck` is a linter that checks for a broader set of errors than `go vet` or `golint`.                                                                                         |
```

Comment:
Nit: do you specifically use megacheck? The ci runs golangci-lint, but I don't believe it runs megacheck

Reply:
I moved this PR over to here: https://github.com/Azure/azure-sdk-for-go/pull/15699

In that branch we don't have this CONTRIBUTING.md anymore since, I believe, we have a repo-wide version of this file.